lib/api/core/restrictedStorage.js

```diff
@@ -180,4 +195,4 @@ class InternalStorage {
   // @todo createCollection + add to indexCache
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)